show/main.py

```diff
@@ -1624,7 +1624,10 @@ def remove_prefix(text, prefix):
         entry = db.get_all(db.STATE_DB, tk)
         r = []
         r.append(remove_prefix(tk, prefix))
-        r.append(entry['restore_count'])
+        if 'restore_count' not in  entry:
```

Comment:
extra space before entry

Reply:
Copied from next if block :-) Will remove it.

Reply:
Done. Removed both.